aptos-move/e2e-move-tests/src/tests/transaction_fee.rs

```diff
@@ -62,3 +105,84 @@ fn test_fee_collection_and_distribution_for_burn_percentages() {
         test_fee_collection_and_distribution(burn_percentage)
     }
 }
+
+#[test]
+fn test_end_to_end_flow() {
+    // Set-up.
+    let mut harness = MoveHarness::new();
+
+    // Dummy accounts.
+    let alice = harness.new_account_at(AccountAddress::from_hex_literal("0xa11ce").unwrap());
+    let bob = harness.new_account_at(AccountAddress::from_hex_literal("0xb0b").unwrap());
+
+    // Initialize core resources and a validator.
+    let core_resources =
+        harness.new_account_at(AccountAddress::from_hex_literal("0xA550C18").unwrap());
+    let validator = harness.new_account_at(AccountAddress::from_hex_literal("0x123").unwrap());
+    let rewards_per_epoch = 285;
+    let mut stake_amount = 25_000_000;
+    assert_success!(setup_staking(&mut harness, &validator, stake_amount));
+    harness.new_epoch();
+
+    // Create transactions to initialize resources for gas fees collection &
+    // distribution and enable/disable it.
+    let burn_percentage = 0;
+    let init_script = create_script(
+        &mut harness,
+        "initialize_collection",
+        &core_resources,
+        vec![TransactionArgument::U8(burn_percentage)],
+    );
+    let enable_script = create_script(&mut harness, "enable_collection", &core_resources, vec![]);
+    // let disable_script = create_script(&mut harness, "disable_collection", &core_resources, vec![]);
```

Comment:
Is this supposed to be in a different test?

Reply:
Yes, fixed